docs/release-notes/deprecate-AgentReattachedEnabled.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Removed AgentReattachedEnabled for Kubernetes, is set always true as of Determined version 0.26.0
+   (Aug 2023).
```

Comment:
[rewrite, apply standards, apply consistency with similar notes]

[e.g., present tense]

Kubernetes: Remove ``AgentReattachedEnabled`` is always true.